dev-ml/camlp5/camlp5-7.10.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib findlib eutils
+
+MY_P=${P%_p*}
+echo $MY_P
```

Comment:
These are probably just for debugging purposes only, but you should take them out before pushing.